share/spice/news/news.js

```diff
@@ -77,22 +77,24 @@
 
         var searchTerm = DDG.get_query().replace(/(?: news|news ?)/i, '').trim();
 
-        if (api_result.error || goodStories >= 3) {
+        if (api_result.error || goodStories < 3) {
```

Comment:
Does `api_result.error` actually show up when there's an error? It looks like it just returns `Spice.failed` when you put in random stuff. https://duckduckgo.com/js/spice/news/acsd%20asdfasdfasdf